src/main.c

```diff
@@ -0,0 +1,60 @@
+#include <clang-c/Index.h>
+#include <stdio.h>
+
+void printCXString(CXString s)
+{
+  printf("%s", clang_getCString(s));
+  clang_disposeString(s);
+}
+
+void get_cursor_type(CXCursor cursor){
```

Comment:
I would rename the prefix of these functions to `print_`... instead of `get_`.

Reply:
Done.